lib/v8-to-istanbul.js

```diff
@@ -51,7 +51,29 @@ module.exports = class V8ToIstanbul {
     if (this.rawSourceMap) {
       if (this.rawSourceMap.sourcemap.sources.length > 1) {
         this.sourceMap = await new SourceMapConsumer(this.rawSourceMap.sourcemap)
-        this.covSources = this.sourceMap.sourcesContent.map((rawSource, i) => ({ source: new CovSource(rawSource, this.wrapperLength), path: this.sourceMap.sources[i] }))
+        // https://github.com/istanbuljs/v8-to-istanbul/issues/166
+        if (!this.sourceMap.sourcesContent) {
+          this.sourceMap.sourcesContent = []
+          for (const relativePath of this.sourceMap.sources) {
+            const fileDir = resolve(this.path, '..')
+            const realPath = resolve(fileDir, relativePath)
```

Comment:
How about integrate with `this._resolveSource` 👀?

Reply:
@bvanjoi This still applies.

https://github.com/istanbuljs/v8-to-istanbul/blob/2f662bf172b3ebe8c04abf4cf5a9297ad7761d6b/lib/v8-to-istanbul.js#L114-L127

you can see `_resolveSource` has a lot of lines that deal with `sourceRoot`, `file://`, `webpack://`, etc. It's absolutely better to use it here. Give it a try?

```js
const realPath = this._resolveSource(this.rawSourceMap, relativePath)
```